docker/src/main/docker/Dockerfile.linux.multistage

```diff
@@ -0,0 +1,21 @@
+## Stage 1 : cope all later needed files from image with native capabilities
```

Comment:
The first stage doesn't seem to do anything?

Reply:
not needed anymore